proxy/src/main/java/org/apache/rocketmq/proxy/processor/ReceiptHandleProcessor.java

```diff
@@ -175,9 +176,10 @@ protected CompletableFuture<MessageReceiptHandle> startRenewMessage(MessageRecei
                 return CompletableFuture.completedFuture(null);
             }
             if (current - messageReceiptHandle.getConsumeTimestamp() < proxyConfig.getRenewMaxTimeMillis()) {
+                RetryPolicy renewPolicy = new RenewStrategyPolicy();
```

Comment:
Changing the RenewStrategyPolicy to `static final` can be better

Reply:
> Changing the RenewStrategyPolicy to `static final` can be better

Thanks，This problem has been modified.